app/controllers/authenticate_controller.rb

```diff
@@ -155,6 +172,35 @@ def handle_authentication_error(err)
     end
   end
 
+  def status_success_json
+    claims = {
+      status: "ok"
+    }
+
+    render json: JSON[claims]
+  end
+
+  def status_failure_json(error)
+    claims = {
```

Comment:
I do remember changing this with @orenbm but it could be that we were focused on the next comment b/c it had to do with the same area and forgot to get back to this. Oren, can you check if this change was made locally?

Reply:
yep, we changed it in the success method and removed the whole method. probably forgot it here.